docs/src/main/asciidoc/cassandra.adoc

```diff
@@ -14,7 +14,7 @@ high availability with no single point of failure.
 
 In this guide, we will see how you can get your REST services to use a Cassandra database.
 
-include::./platform-include.adoc[]
+include::_includes/platform-include.adoc[]
```

Comment:
```suggestion
include::{includes}/platform-include.adoc[]
```

Reply:
Fixed it together with a few other occurrences.